documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -9,243 +9,204 @@ You may obtain a copy of the License at
+|                 |                                                                   | Returns _True_ if the Stream was deleted.                                                                                                               |
+
+
+
+The execution of API `createScope(scope)` establishes that the Scope exists. Then we can create the Stream using the API `createStream(scope, streamName, streamConfig)`. The `StreamManager` requires three parameters to create a Stream:
+
+- Scope Name.
+- Stream Name.
+- Stream Configuration.
+
+The most interesting task is to create the Stream Configuration (`streamConfig`). Like many objects in Pravega, a Stream takes a configuration object that allows
 a developer to control various behaviors of the Stream.  All configuration
-objects in Pravega use a builder pattern for construction.  There are really two
-important configuration items related to streams: Retention Policy and Scaling
-Policy.  
-
-Retention Policy allows the developer to control how long data is kept in a
-Stream before it is deleted.  S/he can specify data should be kept for a certain
-period of time (ideal for situations like regulatory compliance that mandate
-certain retention periods) or to retain data until a certain number of bytes
-have been consumed.  At the moment, Retention Policy is not completely
-implemented.  By default, the RetentionPolicy is set as "unlimited" meaning, data
-will not be removed from the Stream.
-
-Scaling Policy is the way developers configure a Stream to take advantage
-Pravega's auto-scaling feature.  In line 6, we use a fixed policy, meaning the
-Stream is configured with the given number of Stream Segments and that won't
-change.  The other options are to scale by a given number of Events per second
-or a given number of Kilobytes per second.  In these two policies, the developer
-specifies a target rate, a scaling factor and a minimum number of Segments.  The
-target rate is straight forward, if ingest rate exceeds a certain number of
-Events or Kilobytes of data for a sustained period of time, Pravega will attempt
-to add new Stream Segments to the Stream.  If the rate drops below that
-threshold for a sustained period of time, Pravega will attempt to merge adjacent
-Stream Segments.  The scaling factor is a setting on the Scaling Policy that
-determines how many Stream Segments should be added when the target rate (of
-Events or Kilobytes) is exceeded.  The minimum number of Segments is a factor
-that sets the minimum degree of read parallelism to be maintained; if this value
-is set at 3, for example, there will always be 3 Stream Segments available on
-the Stream.  Currently, this property is effective only when the stream is
+objects in Pravega use a builder pattern for construction. [**Retention Policy**](pravega-concepts.md#retention-policy) and [**Scaling
```

Comment:
I would change a bit the sentences here: `...configuration object instantiated via builder pattern. That allows a developer to control various aspects of a Stream's behavior in terms of _policies_; Retention policies and Scaling policies are the most important ones related to Streams.`
Note that I have not added the links, but I assume them to be as in the previous text block.

Reply:
Accepted suggestion